theme/static/js/custom.js

```diff
@@ -171,26 +171,42 @@ $(document).ready(function () {
         return false
     });
 
-    $.ajax({
-        url: "/hsapi/userInfo/",
-        // success: function(user) {
-        //     if(!user.organization) {
-        //         // Disable publishing resources
-        //         if ($("#publish").length) {
-        //             $("#publish").toggleClass("disabled", true);
-        //             $("#publish").removeAttr("data-toggle");   // Disable the agreement modal
-        //             $("#publish > [data-toggle='tooltip']").attr("data-original-title",
-        //                 "Your profile information must be complete before you can formally publish resources.");
-        //         }
-        //
-        //         var message = 'Your profile is nearly complete. Please fill in the '
-        //             + '<strong>Organization</strong> field'
-        //             + ' on the <a href="/user/' + user.id + '/">User Profile</a> page';
-        //         showUniversalMessage("warn", message, 10000)();
-        //     }
-        // },
-        error: showUniversalMessage()
-    })
+    if (window.location.hash == '#_'){
+        $.ajax({
+            url: "/hsapi/userInfo/",
+            success: function(user) {
+
+                var missing_fields = []
+                for(key in user){
+                    if(!user[key]){
+                        missing_fields.push(key)
+                    }
```

Comment:
The fields displayed here are not human readable fields (`last_four_ss` vs `Last four digits of social security number`), and I think they should be given that they're presented to the user. The first thing that I would think to do in this kind of scenario (need to use the same human readable text both in django templates, and in javascript client side code), would be to use the django internationalization support (https://docs.djangoproject.com/en/1.8/topics/i18n/translation/#internationalization-in-javascript-code), then use the same i18n keys to lookup what the human value would be in django templates, and on the js side.

But...i18n isn't turned on in this project. I'm tempted to say don't show which fields are missing :/